filer/static/filer/js/base.js

```diff
@@ -43,34 +43,34 @@ Cl.mediator = new Mediator();
         }
 
         $('.js-filter-files').on('focus blur', function (event) {
-            var container = $(this).closest('.js-filter-files-container');
+            var container = $(this).closest('.navigator-top-nav');
```

Comment:
@vxsx it should called filter because filter in header is meant. Should i change to filer-filter..?